.travis.yml

```
@@ -32,6 +32,9 @@ script:
   - black --check .
   - flake8 --config=.flake8 .
   - mypy cgp
+  - if [ "$DEP" = "[all]" ]; then
+       cd docs && make html && cd ../ || exit 1;
```

```suggestion
       cd docs && make html || exit 1;
```
you don't need to go back, do you?

---

not sure, don't we need to be in the parent directory for the following steps?

---

i think after the command you're anyway in the directory where you started from

---

I think, we need the `cd ../` here. After removing it, the tests were not found, I guess, because we're not in the parent directory.